common/src/Module.php

```diff
@@ -175,7 +161,27 @@ public function hookHeader($params) {
         ]);
     }
 
-    public function hookFooter($params) {
+
+    public function hookDisplayOrderConfirmation($params) {
+        $order = $params['order'];
+        $customer = new \Customer((int) ($order->id_customer));
```

Comment:
```suggestion
        $customer = new \Customer((int) $order->id_customer);
```

~~~
> $x = (object) ['y' => '1'];
= {#5209
    +"y": "1",
  }

> (int) $x->y;
= 1

> (int) ($x->y);
= 1
~~~

Reply:
ah I am blind ... I did not notice the `()`